src/core/components/rerender/Rerender.test.tsx

```diff
@@ -0,0 +1,46 @@
+import '@testing-library/jest-dom';
+import { act, render, screen } from '@testing-library/react';
+import { Rerender } from './Rerender';
+
+jest.useFakeTimers();
+
+describe('<Rerender /> component', () => {
+    it('renders the initial time', () => {
```

Comment:
Could you add a `createTestComponent` function as done for all other component tests? It helps sharing the logic of rendering a component and is useful especially when adding a new required property to a component to avoid updating the usage across all tests

Reply:
✅